### PR TITLE
Just a small modification

### PR DESCRIPTION
**Pull request by melcsi**

### CODE REVIEW

**Comment by melcsi** [comment]
Comment is not properly added

File: src/HelloWorld.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// modified by melcsi
+
```
Comment (melcsi) [inline src/HelloWorld.java]: This comment is wrong

File: src/HelloWorld.java [hunk 2]
```diff
@@ -1,3 +1,5 @@
+// modified by melcsi
```
Comment (godri) [inline src/HelloWorld.java]: A timestamp should be added

Comment (melcsi) [inline src/HelloWorld.java]: OK

**Pull request merged**